### PR TITLE
fix: Added the following changes:

### PR DESCRIPTION
**Pull request by ravali-rimmalapudi**
1. Updated OAI commit not to include in release notes and changeLog
2. Updated the yml with proper names

Closes: [DII-133](https://issues.corp.twilio.com/browse/DII-133)
Dependent PR: [CLI-CORE](https://github.com/twilio/twilio-cli-core/pull/161)

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

**Comment by ravali-rimmalapudi** [comment]
Successful workflow Runs:
CLI: https://github.com/ravali-rimmalapudi/twilio-cli/actions/runs/1294477470
Oclif: https://github.com/ravali-rimmalapudi/twilio-cli/actions/runs/1294488415
Platform executables: https://github.com/ravali-rimmalapudi/twilio-cli/actions/runs/1294488205
Release tag: https://github.com/ravali-rimmalapudi/twilio-cli/releases/tag/2.30.3

**Review by kridai — APPROVED** [review]

File: .releaserc.json [hunk 1]
```diff
@@ -14,6 +14,12 @@
         "releaseRules": [{
           "type": "chore",
           "release": "patch"
+        },{
+          "type": "oaiFix",
```
Comment (shrutiburman) [inline .releaserc.json:18]: Does this mean commits with type oaiFix would go in `patch` realease and so on? 
Also, for commits, in general, should we follow a format? Something like <type>:<message> ?

**Review by shrutiburman — APPROVED** [review]
LGTM!

**Review by shamantraghav — COMMENTED** [review]
LGTM!

**Review by onuzbee — COMMENTED** [review]
LGTM! 
Please change the commit title to reflect change crisply in a oneline format

**Pull request merged**